src/test/java/com/thealgorithms/sorts/Sum_Of_SubsetTest.java

```diff
@@ -0,0 +1,21 @@
+package com.thealgorithms.sorts;
```

Comment:
This should fix the failing build:

```suggestion
package com.thealgorithms.dynamicprogramming;
```

Reply:
done